spec/controllers/hub/clients_controller_spec.rb

```diff
@@ -734,9 +736,9 @@
   end
 
   describe "#update_take_action" do
-    let(:user) { create :user, vita_partner: (create :vita_partner) }
+    let(:user) { create :user, vita_partner: (create :organization) }
```

Comment:
Probably don't need to set the `vita_partner_id` field on `user` here. Non-blocking.